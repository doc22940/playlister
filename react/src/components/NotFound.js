import React, { Component } from 'react';

import gif from '../media/cat.gif';

class NotFound extends Component {
  render() {
    return (
      <div className='jumbotron justify-content-middle text-center'>
        <p className='lead h3'>We're trying to find what you are looking for... but it is proving difficult...</p>
        <img src={gif} alt='Fat Cat' className='img-fluid' style={{ width: '80%' }} />
      </div>
    );
  }
}

export default NotFound;
