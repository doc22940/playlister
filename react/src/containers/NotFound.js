import React, { Component } from 'react';

import gif from '../media/cat.gif';

class NotFound extends Component {
  render() {
    return (
      <div className='jumbotron justify-content-middle text-center'>
        <h1 style={{ fontSize: '150px' }}>404</h1>
        <p className='lead'>We're trying to find what you are looking for... but it is proving difficult...</p>
        <img src={gif} alt='Fat Cat' className='img-fluid' style={{ width: '80%' }} />
      </div>
    );
  }
}

export default NotFound;
