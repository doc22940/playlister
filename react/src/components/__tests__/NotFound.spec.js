import React from 'react';
import renderer from 'react-test-renderer';

import NotFound from '../NotFound.js';

test('NotFound snapshot', () => {
  const component = renderer.create(
    <NotFound />,
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
