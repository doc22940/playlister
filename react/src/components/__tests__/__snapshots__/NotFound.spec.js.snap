// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound snapshot 1`] = `
<div
  className="jumbotron justify-content-middle text-center"
>
  <p
    className="lead h3"
  >
    We're trying to find what you are looking for... but it is proving difficult...
  </p>
  <img
    alt="Fat Cat"
    className="img-fluid"
    src="cat.gif"
    style={
      Object {
        "width": "80%",
      }
    }
  />
</div>
`;
